build.gradle.kts

```diff
@@ -81,23 +83,75 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:${properties["coroutinesVersion"]}")
     implementation("org.postgresql:postgresql:${properties["driverVersions.postgres"]}")
 
-
-    testImplementation("io.mockk:mockk:1.13.5")
-    testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.0")
-    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.8.0")
-    testImplementation("net.bytebuddy:byte-buddy:1.14.6")
-    testImplementation("net.bytebuddy:byte-buddy-agent:1.14.6")
+    // Updated test dependencies for Gradle 9 compatibility with logging fixes
+    testImplementation("io.mockk:mockk:${properties["mockkVersion"]}") {
+        exclude(group = "org.slf4j", module = "slf4j-api")
+    }
+    testImplementation("org.junit.jupiter:junit-jupiter-api:${properties["junitVersion"]}")
+    testImplementation("org.junit.jupiter:junit-jupiter-params:${properties["junitVersion"]}")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${properties["junitVersion"]}")
```

Comment:
Why was the library "org.junit.jupiter:junit-jupiter-params:${properties["junitVersion"]}" added?

Reply:
removed